frontend/src/features/properties/map/acquisition/add/AddAcquisitionContainer.tsx

```diff
@@ -33,7 +62,7 @@ export const AddAcquisitionContainer: React.FC<IAddAcquisitionContainerProps> =
     history.replace(`/mapview/sidebar/acquisition/${acqFile.id}`);
   };
 
-  const helper = useAddAcquisitionFormManagement({ onSuccess });
+  const helper = useAddAcquisitionFormManagement({ onSuccess, initialForm });
```

Comment:
this name does not give much information. Can the logic from the hook be put here or is reused somewhere else?

Reply:
This comes down to developer preference. I like to "hide" business logic in forms in single-use hooks. Looks cleaner to me rather than inlining 20 lines of code but I'm open to hear if others dislike that approach and we can decide as a team what we prefer. These hooks are not meant to be reused but rather to hide complexity from a rather large form. I try to think of them as "helpers" or static classes in C#